scripts/speaker_tasks/scp_to_manifest.py

```diff
@@ -56,7 +57,7 @@ def filter_manifest_line(manifest_line, signal, sr=16000):
         remaining_dur = remaining_dur - temp_dur
         while remaining_dur >= 0:
             segment_audio = signal[int(start * sr) : int(start * sr + temp_dur * sr)]
-            if l.feature.rms(segment_audio).mean() > 0.01:
+            if l.feature.rms(y=segment_audio).mean() > 0.01:
```

Comment:
The best practice is avoiding hard-coded numbers in the input statement, since it makes hard to debug or re-use the function.  Not just this, dur > 1.5: ([this line](https://github.com/NVIDIA/NeMo/blob/6bc6a05f2f92d4e50cfe5399bf56290a3082f580/scripts/speaker_tasks/scp_to_manifest.py#L54))is better to be changed

Reply:
Done